src/ContainerInstance/help/New-AzContainerInstanceObject.md

```diff
@@ -56,6 +56,16 @@ test-container
 
 Create a container instance using image alphine with limit cpu 2.0 and limit memory 2.5Gb
 
+### Example 3: Create a container group with a container instance
+```powershell
+PS C:\> $container = New-AzContainerInstanceObject -Name test-container -Image alpine
+PS C:\> New-AzContainerGroup -ResourceGroupName testrg-rg -Name test-cg -Location eastus -Container $container
+
+Location Name    Zone ResourceGroupName
```

Comment:
Hi @CaptainFanZzz , thanks a lot for your contribution! We have example of `Create container group with ContainerInstanceObject` in `New-AzContainerGroup.md`. I will close this PR. Thanks all the same.

Reply:
Hi @wyunchi-ms , this is an improvement due to feedback from customer, so I reopen it. Customer may not understand it when only seeing this cmdlet page. Thanks.

Reply:
Then you need to update it in generation branch. It will be force cover when sync up with generation branch.

Reply:
We manually change both sides because it's just the update of doc, so it's unnecessary to run pipeline from generation to sync with main branch. Thanks:)